### PR TITLE
NEW #3509 payment ref for suppliers

### LINKED ISSUE #3509: Add a numbering module for payments
So payment will have ref = "PAYYYMM00001" instead of 1, 2, 3...
- [x] Payment ref for customers
- [ ] Payment ref for suppliers

### PR DESCRIPTION
**Pull request by aconits**
I added the part of supplier payment, I hope that I did it well :smile: 
@eldy

### CODE REVIEW

File: htdocs/core/lib/fourn.lib.php [hunk 1]
```diff
@@ -178,6 +178,11 @@ function supplierorder_admin_prepare_head()
 	$head[$h][1] = $langs->trans("SuppliersInvoice");
 	$head[$h][2] = 'invoice';
 	$h++;
+	
+	$head[$h][0] = DOL_URL_ROOT."/admin/supplier_payment.php";
+	$head[$h][1] = $langs->trans("SuppliersPayment");
+	$head[$h][2] = 'supplierpayment';
+	$h++;
```
Comment (eldy) [inline htdocs/core/lib/fourn.lib.php:190]: Payments are related to invoice. Why not adding option of payment into existing tab  /admin/supplier_invoice.php ?

**Comment by eldy** [comment]
This looks to be a new feature, not a fix ?

**Comment by aconits** [comment]
Yes it's look new feature, I tag it as fix because I make it from issue #3509

File: htdocs/core/lib/fourn.lib.php (hunk 1 above)
Comment (aconits) [inline htdocs/core/lib/fourn.lib.php:190]: I make it fast by coping the customer payment system

**Comment by eldy** [comment]
OK. Great. This PR is really welcome. It will open road to mutualize some code. However, can you send it to develop branch instead of 3.9 ?

**Comment by aconits** [comment]
My bad again I'm thought it was for 3.9
I will move it on develop

**Pull request closed without merging**